### PR TITLE
Move Linux builds to CircleCI

### PR DESCRIPTION
**Pull request by sas**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -26,9 +26,16 @@ build_step: &build_step
 test_step: &test_step
   run:
     name: Test
-    command: test -z "$LLDB_TESTS" || (cd build && ../Support/Scripts/run-lldb-tests.sh)
+    command: cd build && ../Support/Scripts/run-lldb-tests.sh
```
Comment (xiaobai) [inline .circleci/config.yml]: why no more test?

Comment (xiaobai) [reply .circleci/config.yml]: Oh wait, because we actually want to perform the tests now?

**Comment by sas** [comment]
Yeah. Mostly because it’s cleaner in that format.
-- 
-- 
Stephane Sezer

File: .circleci/config.yml (hunk 1 above)
Comment (sas) [reply .circleci/config.yml]: Yeah we include the test step only when we actually want to run tests.

**Pull request merged**